lib/modelMaterialsCommon.js

```diff
@@ -727,6 +729,85 @@ function lightDefaults(gltf) {
     }
 }
 
+function getShaderVariable(accessor) {
```

Comment:
Can you update the spacing in this file to be consistent with the rest of the codebase?